src/CoreBundle/Session/ChameleonSessionManager.php

```diff
@@ -63,17 +64,19 @@ class ChameleonSessionManager implements ChameleonSessionManagerInterface
      * @var bool
      */
     private $isSessionStarting = false;
+    private PortalDomainServiceInterface $portalDomainService;
 
     /**
-     * @param RequestStack             $requestStack
-     * @param Connection               $databaseConnection
-     * @param PDO                      $sessionPdo
-     * @param ContainerInterface       $container
-     * @param int                      $metaDataTimeout
-     * @param array                    $sessionOptions
-     * @param InputFilterUtilInterface $inputFilterUtil
+     * @param RequestStack                  $requestStack
```

Comment:
I know this is just a reformat, but this docblock adds 0 value to the code since it just repeats the type hints. It could be narrowed down to just one type, which should be more clearly defined though: The array format of `$sessionOptions` (Which is `array<string, mixed>` I suppose?)

Reply:
Yes, I thought about that for a second but then forgot. Done.

Reply:
Ah, we missed `$metaDataTimeout` which does not have a type annotation - it is also required in the docblock

Reply:
done